packages/sonarwhal/package.json

```diff
@@ -24,6 +24,7 @@
     "globby": "^8.0.1",
     "handlebars": "^4.0.11",
     "iconv-lite": "^0.4.19",
+    "iltorb": "^2.0.6",
```

Comment:
Is this faster then https://www.npmjs.com/package/brotli? Or what is the reasoning behind choosing it?

Reply:
IIRC this is the same one we are using in the compression rule and it's a native implementation.

Reply:
> IIRC this is the same one we are using in the compression rule and it's a native implementation.

See: https://github.com/sonarwhal/sonarwhal/blob/master/packages/rule-http-compression/package.json#L10

Reply:
I've done a bit of research and I think we should switch the compression rule to `iltorb` even though it's a binary one. The reason is that [`brotli` hasn't been updated in about a year](https://github.com/devongovett/brotli.js/commits/master) and new versions of the algorithm have been released. The maintainers of `iltorb` seem to be doing a [good job at having everything up to date](https://github.com/MayhemYDG/iltorb/blob/master/CHANGELOG.md).

Reply:
> I've done a bit of research and I think we should switch the compression rule to iltorb even though it's a binary one. The reason is that brotli hasn't been updated in about a year and new versions of the algorithm have been released. The maintainers of iltorb seem to be doing a good job at having everything up to date.

SGTM!